keyboards/und/pcm-t-k0-windows.xml

```diff
@@ -34,7 +34,7 @@
 		<map iso="D10" to="p"/>
 		<map iso="D11" to="["/>
 		<map iso="D12" to="]"/>
-		<map iso="D13" to="/"/>
+		<map iso="C12" to="/"/>
```

Comment:
Please move this down to the appropriate place for C12 to avoid confusion in the future.

Reply:
OK